.github/workflows/dialogflow-cx-python.yaml

```diff
@@ -0,0 +1,173 @@
+          workload_identity_provider: 'projects/792212140555/locations/global/workloadIdentityPools/gh-pool-df-3/providers/gh-provider-df-3'
+          service_account: 'oidc-sa-df-3@df-terraform-dev.iam.gserviceaccount.com'
+      - id: 'basic_sample'
+        name: Get Webhook Env Vars
+        run: |-
+          export WEBHOOK_FUNCTION_NAME 
+          export WEBHOOK_FUNCTION_ENTRYPOINT 
+          WEBHOOK_FUNCTION_NAME="$(python -c "from webhook.main import get_webhook_name; print(get_webhook_name(\"${BUILD_UUID?}\"))")"
+          WEBHOOK_FUNCTION_ENTRYPOINT="$(python -c "from webhook.main import get_webhook_entrypoint; print(get_webhook_entrypoint())")"
+          echo "::set-output name=webhook-name::$WEBHOOK_FUNCTION_NAME"
+          echo "::set-output name=webhook-entrypoint::$WEBHOOK_FUNCTION_ENTRYPOINT"
+        env:
+          PROJECT_ID: df-terraform-dev
+          BUILD_UUID: ${{ steps.build_id.outputs.id }}
+      - name: Terraform Init/Apply
+        run: |-
+          cp "${{ steps.auth.outputs.credentials_file_path }}" terraform/application_default_credentials.json
+          cp -r webhook terraform
+          docker run --env GOOGLE_APPLICATION_CREDENTIALS=/app/application_default_credentials.json -w /app -v "$(pwd)"/terraform:/app -v "$(pwd)"/webhook:/app/webhook hashicorp/terraform:1.1.7 init
+          docker run --env GOOGLE_APPLICATION_CREDENTIALS=/app/application_default_credentials.json -w /app -v "$(pwd)"/terraform:/app -v "$(pwd)"/webhook:/app/webhook hashicorp/terraform:1.1.7 apply --auto-approve -var project_id="${PROJECT_ID?}" -var webhook_function_name="${WEBHOOK_FUNCTION_NAME?}" -var webhook_function_entrypoint="${WEBHOOK_FUNCTION_ENTRYPOINT?}"
+        env:
+          PROJECT_ID: df-terraform-dev
+          WEBHOOK_FUNCTION_NAME: ${{ steps.basic_sample.outputs.webhook-name }}
+          WEBHOOK_FUNCTION_ENTRYPOINT: ${{ steps.basic_sample.outputs.webhook-entrypoint }}
+  integration_tests:
+    needs: [setup_terraform]
+    if: ${{ success() }}
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./dialogflow-cx
+    strategy:
+      matrix:
+        python: ["3.9"]
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python }}
+      - uses: actions/checkout@v3
+      - name: Install requirements.txt
+        run: |-
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -r requirements-test.txt
+      - id: 'auth'
+        name: 'Authenticate to GCP'
+        uses: 'google-github-actions/auth@v0'
+        with:
+            create_credentials_file: 'true'
+            workload_identity_provider: 'projects/792212140555/locations/global/workloadIdentityPools/gh-pool-df-3/providers/gh-provider-df-3'
+            service_account: 'oidc-sa-df-3@df-terraform-dev.iam.gserviceaccount.com'
```

Comment:
Another potential candidate for an env variable

Reply:
Done.